addon/components/async-button.js

```diff
@@ -8,7 +8,8 @@ const {
   observer,
   deprecate,
   getWithDefault,
-  Component
+  Component,
+  Object:EmObject
```

Comment:
needs a space after the colon, and DockYard prefers `EmberObject` over `EmObject`